### ISSUE TITLE
Media title is not matching in gallery, when swiping

### ISSUE THREAD

**Issue by coone4life** [nextcloud/android#13163]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

Open app, open media, open a media file and have a look at the name of the file on top of screen, swipe to one side and you see the name is not switching to the name of the actual media file.

### Expected behaviour

Should show the name of the displayed media file

### Actual behaviour

Shows the file name of the first opened picture

### Android version

14

### Device brand and model

Pixel 7 Pro

### Stock or custom OS?

Stock

### Nextcloud android app version

3.29.1 RC1

### Nextcloud server version

29.0.2

### Using a reverse proxy?

No

### Additional information

So in the the QA app Version, that APK I can download from this thread #13132,  it works perfectly. 
But in the official app the name of the file does not match with the actual showed picture

**Comment by falolaf** [comment]
I noticed this today. Nextcloud 3.29.2. Samsung Galaxy Tab S9 and Galaxy S21.

**Comment by PhilLab** [comment]
I assume this has been fixed with https://github.com/nextcloud/android/pull/15669 (v 3.35) and https://github.com/nextcloud/android/pull/16297 (v 33). If you don't mind, please retest with the latest version.

**Comment by falolaf** [comment]
Yes, seems to be working for me. Hadn't thought about this for quite a while.
